doc/user/content/concepts/_index.md

```diff
@@ -18,6 +18,7 @@ The pages in this section introduces some of the key concepts in Materialize:
 
 Component                                | Use
 -----------------------------------------|-----
+[Reaction Time](/concepts/reaction-time) | The ability to react to real-world changes.
```

Comment:
So ... the "Reaction time" is "The ability to ..."  ?  

Instead, would probably mention in the description something w.r.t. freshness and query latency as the reaction time.

(And if you don't mind ... if we could change the table columns to "Concept" and "Description" ... I had just ported the table from an old page and didn't update the column names 😞 )